### PR TITLE
Rename project

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues:

* [FH-4418](https://issues.jboss.org/browse/FH-4418) - Remove mcp from package in fh-android-core
* [FH-4419](https://issues.jboss.org/browse/FH-4419) - Remove .idea folder from fh-android-core

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun Can you double check it?

**Comment by secondsun** [comment]
Remove the ExampleUnitTest and LGTM

File: library/src/test/java/org/feedhenry/core/ExampleUnitTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.feedhenry.mcp.mobile_core;
+package org.feedhenry.core;
```
Comment (secondsun) [inline library/src/test/java/org/feedhenry/core/ExampleUnitTest.java]: File can probably be removed

Comment (secondsun) [reply library/src/test/java/org/feedhenry/core/ExampleUnitTest.java]: This test file I mean.

Comment (danielpassos) [reply library/src/test/java/org/feedhenry/core/ExampleUnitTest.java]: Done!

**Pull request merged**